microsoftgraph/microsoft-graph-explorer-v2
# enable sending of request headers with query

## Overview

Enables the sending of request headers with the queries

### Demo

Optional. Screenshots, `curl` examples, etc.

### Notes

Does not remove the header from the display but removes it from the 'behind the scenes' array

## Testing Instructions

* Head over to the request headers tab
* Put in the header name in the key field and the header value in the key field
* hit run query
* Investigate using the developer console the request headers that you have sent out. The ones you added will appear
